src/python/WMCore/MicroService/Unified/MSOutput.py

```diff
@@ -0,0 +1,170 @@
+        self.msConfig.setdefault("defaultGroup", "DataOps")
+        self.msConfig.setdefault("enableAggSubscr", True)
+        self.msConfig.setdefault("enableDataPlacement", False)
+        self.msConfig.setdefault("excludeDataTier", ['NANOAOD'])
+        self.msConfig.setdefault("rucioAccount", 'wma_test')
+        self.uConfig = {}
+        self.emailAlert = EmailAlert(self.msConfig)
+
+    @retry(tries=3, delay=2, jitter=2)
+    def updateCaches(self):
+        """
+        Fetch some data required for the output logic, e.g.:
+        * unified configuration
+        """
+        self.uConfig = self.unifiedConfig()
+        if not self.uConfig:
+            raise RuntimeWarning("Failed to fetch the unified configuration")
+
+    def execute(self, reqStatus):
+        """
+        Executes the whole output data placement logic
+        :return: summary
+        """
+        # start threads in MSManager which should call this method
+        # NOTE:
+        #    Here we should make the whole logic - like:
+        #    * Calling the system to fetch the workflows from;
+        #    * Creating the workflow objects;
+        #    * Pushing them into the back end database system we choose for bookkeeping
+        #    * Updating their status in that system, both MsStatus (subscribed,
+        #      processing, etc.) and also the Reqmgr status
+        #    * Associate and keep track of the requestID/subscriptionID/ruleID
+        #      returned by the Data Management System and the workflow
+        #      object (through the bookkeeping machinery we choose/develop)
+        summary = dict(OUTPUT_REPORT)
+
+        # We should decide here what variable name to use request vs. workflow
+        try:
+            requestRecords = self.getRequestRecords(reqStatus)
+            self.updateReportDict(summary, "total_num_requests", len(requestRecords))
+            msg = "  retrieved %s requests. " % len(requestRecords)
+            msg += "Service set to process up to %s requests per cycle." % self.msConfig["limitRequestsPerCycle"]
+            self.logger.info(msg)
+        except Exception as err:  # general error
+            msg = "Unknown exception while fetching requests from ReqMgr2. Error: %s", str(err)
+            self.logger.exception(msg)
+            self.updateReportDict(summary, "error", msg)
+
+        try:
+            self.updateCaches()
+        except RuntimeWarning as ex:
+            msg = "All retries exhausted! Last error was: '%s'" % str(ex)
+            msg += "\nRetrying to update caches again in the next cycle."
+            self.logger.error(msg)
+            self.updateReportDict(summary, "error", msg)
+            return summary
+        except Exception as ex:
+            msg = "Unknown exception updating caches. Error: %s" % str(ex)
+            self.logger.exception(msg)
+            self.updateReportDict(summary, "error", msg)
+            return summary
+
+        # this one is put here just for example.
```

Comment:
Then I suggest to always mark it as TODO or FIXME to get it highlighted in the code and in pylint